package.json

```diff
@@ -8,8 +8,9 @@
     "test": "sk-tests",
     "test/watch": "sk-tests-watch",
     "test/perf": "karma start --single-run --perf",
-    "lint": "eslint -c node_modules/skatejs-build/.eslintrc src",
-    "prepublish": "sk-bundle"
+    "lint": "eslint -c node_modules/skatejs-build/.eslintrc src, test --ext .js,.jsx --format 'node_modules/eslint-friendly-formatter'",
```

Comment:
the `named-slots/.eslintrc` file pulls in `skatejs-build/.eslintrc`, so we probably don't need to directly reference `node_modules/skatejs-build/.eslintrc` here. up to you :)